src/index.js

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import TodoApp from 'components/TodoApp';
 import './styles/app.scss';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
- [ ] Excellent job implementing Routs and Router in your index.js, however, it is a good practice to import all objects before importing the styles files, kindly consider exchanging line 4  `import './styles/app.scss';` for line 5  `import { BrowserRouter as Router } from 'react-router-dom';`